server/server.go

```diff
@@ -44,9 +44,13 @@ type RPCServer struct {
 	// url is the URL where the server should dial to start subscribing.
 	url string
 
-	// handlers is a map where the routing key is used as map key and the value
-	// is a function of type HandlerFunc.
-	handlers map[string]HandlerFunc
+	// handlers is a list of handlerBinding that holds information about the
+	// bindings and handlers.
+	handlers []handlerBinding
```

Comment:
Perhaps this should be named `bindings`?

Reply:
Sure, that might be a better name.